modules/aws-emr-cluster/variables.tf

```diff
@@ -181,3 +181,9 @@ variable "utility_script_bucket_key" {
   description = "Key (i.e. path) to upload the utility script to"
   default     = "util/upload_hbase_config.sh"
 }
+
+variable "custom_ami_id" {
```

Comment:
I think this would have to be `emr_custom_ami_id` to match up with how you use `var.emr_custom_ami_id` in modules/aws-emr-cluster/main.tf